mapper/src/main/java/org/jboss/pnc/mapper/UrlTreeSetAndOperationParametersConverter.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.mapper;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+/**
+ * Converts between operation parameters, which represent URL and list of URLs.
+ */
+@ApplicationScoped
+public class UrlTreeSetAndOperationParametersConverter {
+
+    private static final String URL_PARAMETER_PREFIX = "url-";
+
+    public static List<String> urlSetFromOperationParameters(Map<String, String> operationParameters) {
+        return operationParameters.entrySet()
+                .stream()
+                .filter(entry -> entry.getKey().startsWith(URL_PARAMETER_PREFIX))
+                .map(Map.Entry::getValue)
+                .distinct()
+                .collect(Collectors.toList());
```

Comment:
Please order it (either alphabetically or by the url-X's number X)

Reply:
[bc70ba0](https://github.com/project-ncl/pnc/commit/bc70ba02ff1dc4d669166ae62ce9861af9a5504c)